requirements.txt

```diff
@@ -0,0 +1 @@
+quart>=0.18.4
```

Comment:
This is the version that is currently the most recent. Should we support old versions, too? I can argue either way: ideally you're running through `poetry` or virtual environments or something, so versioning is not an issue, and this is fine. but in practice many people are not doing that, and they might have old versions of quart installed, which probably work great with this, too, despite what this file says. ...and yet, if you're doing that, you're probably not using this file at all, and you'll just run main.py with an old version of Quart and things will work fine anyway. 

...so, I think I'm leaning towards just leaving this as-is, but I'm open to disagreement. Someone better at modern Python than I am will probably know the right choice here.

Reply:
Like you pointed out, if someone is using an older version of Quart, they're not using this file. The only reason I'd ever see them needing an older version is if they were doing something else that only operates in an old version, which is something I don't think we have to worry about in this case.

Reply:
Like you pointed out, if someone is using an older version of Quart, they're not using this file. The only reason I'd ever see them needing an older version is if they were doing something else that only operates in an old version, which is something I don't think we have to worry about in this case.